OpenRA.Mods.Common/UpdateRules/Rules/20180923/AddRearmable.cs

```diff
@@ -0,0 +1,102 @@
+#region Copyright & License Information
+/*
+ * Copyright 2007-2018 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation, either version 3 of
+ * the License, or (at your option) any later version. For more
+ * information, see COPYING.
+ */
+#endregion
+
+using System.Collections.Generic;
+
+namespace OpenRA.Mods.Common.UpdateRules.Rules
+{
+	public class AddRearmable : UpdateRule
+	{
+		public override string Name { get { return "Added Rearmable trait and move RearmBuildings properties there"; } }
+		public override string Description
+		{
+			get
+			{
+				return "Added Rearmable trait and replaced Aircraft.RearmBuildings and\n" +
+					"Minelayer.RearmBuildings with Rearmable.RearmActors.";
+			}
+		}
+
+		public override IEnumerable<string> UpdateActorNode(ModData modData, MiniYamlNode actorNode)
+		{
+			var aircraftNodes = actorNode.ChildrenMatching("Aircraft");
+			var minelayerNodes = actorNode.ChildrenMatching("Minelayer");
+			var ammoPoolNodes = actorNode.ChildrenMatching("AmmoPool");
+			var addNodes = new List<MiniYamlNode>();
+
+			var ammoPoolNames = new List<string>() { "primary" };
+			foreach (var ap in ammoPoolNodes)
+			{
+				var poolName = ap.LastChildMatching("Name");
+				if (poolName != null && poolName.Value.Value != "primary")
```

Comment:
We usually use `.NodeValue<T>` (`T` being `string` here) instead of `.Value.Value`. (Applies to the line below, too.)